elasticsearch/initialize_es_index.py

```diff
@@ -3,39 +3,58 @@
 1. es_index_prefix (CLI parameters)
 2. type of the data (defined in the global variable TYPES
 """
-
-from elasticsearch import Elasticsearch
 import os
 import click
+from elasticsearch import Elasticsearch
 
-
-TYPES = ["organism", "specimen", "dataset", "experiment", "file"]
+import sys
+sys.path.append('../scripts')
+from constants import INDICES
 
 
 # use click library to get command line parameters
 @click.command()
+@click.argument('es_index_prefix')
 @click.option(
     '--es_host',
     default="http://wp-np3-e2:9200",
     help='Specify the Elastic Search server (port should be included), default to be http://wp-np3-e2:9200.'
 )
 @click.option(
-    '--es_index_prefix',
-    help='Specify the Elastic Search index prefix'
+    '--delete_only',
+    default=False,
+    help='Indicate whether to create empty indices, i.e. if set to True, this scripts turns to delete existing indices'
 )
-def main(es_host, es_index_prefix) -> None:
+def main(es_host, es_index_prefix, delete_only) -> None:
+    """
+    Script to initialize/delete a build of indices determined by parameter es_index_prefix on Elastic Search server
+    if parameter delete_only is true, only delete any existing indices matching the prefix pattern,
+    no new indices will be created
+    """
+    """
+    :param es_host: Elastic
```

Comment:
You need to add documentation to these parameters and delete return if function doesn't return anything.

Reply:
done in bf3d732